api-outbound.yaml

```diff
@@ -120,6 +120,57 @@ paths:
         basicauth:
           username: "${NEXPOSE_API_USERNAME}"
           password: "${NEXPOSE_API_PASSWORD}"
+  /api/3/:
+    get:
+      description: Nexpose API root endpoint, used for verifying if Nexpose can be reached
+      responses:
+        200:
+          description: "Success"
+          content:
+            application/json:
+              schema:
+                type: object
+        401:
+          description: "Unauthorized"
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/NexposeError'
+        404:
+          description: "Not Found"
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/NexposeError'
+      x-transportd:
+        backend: nexpose
+        enabled:
+          - "accesslog"
+          - "responsevalidation"
+          - "timeout"
+          - "retry"
+          - "basicauth"
+        timeout:
+          after: "2s"
```

Comment:
Suggestion: This may be a basic endpoint, but it's still Nexpose. Might want to up this to about 30 seconds, which leaves plenty of time for the check to pass in our own environment.

Reply:
sure